MapboxNavigation/NavigationComponent.swift

```diff
@@ -37,7 +37,11 @@ public protocol CarPlayConnectionObserver: class {
  */
 public protocol NavigationStatusPresenter: class {
     /**
-     Shows the status view for a specified amount of time.
+     Shows a Status for a specified amount of time.
      */
     func showStatus(title: String, spinner: Bool, duration: TimeInterval, animated: Bool, interactive: Bool)
+    
+    func addNewStatus(status: StatusView.Status)
```

Comment:
I'd also rename `addNewStatus(status:)` to `func show(_ status: StatusView.Status)` and `func hideStatus(using: StatusView.Status)` to `hide(_ status: StatusView.Status)`.